feg/gateway/services/swx_proxy/client_api_test.go

```diff
@@ -91,17 +93,21 @@ func standardSwxProxyTest(t *testing.T) {
 		t.Fatalf("GRPC SAR Register Error: %v", err)
 		return
 	}
-	assert.Equal(t, &protos.RegistrationAnswer{SessionId: regRes.GetSessionId()}, regRes)
-	t.Logf("GRPC Register SAA: %#+v", *regRes)
+	equal := proto.Equal(&protos.RegistrationAnswer{SessionId: regRes.GetSessionId()}, regRes)
+	assert.True(t, equal)
```

Comment:
This lines are necessary because of the new protobuf implementation. Messages now contain a `sync.Mutex` such that copying is detected in govet checks. Let me copy an example of such a failing check for clarity:

```
services/tenants/servicers/protected/servicer_test.go:154:18: copylocks: call of assert.Equal copies lock value: magma/orc8r/cloud/go/services/tenants/protos.GetControlProxyResponse contains google.golang.org/protobuf/internal/impl.MessageState contains sync.Mutex (govet)
	assert.Equal(t, *controlProxy, sampleGetControlProxyRes)
```

This can be avoided by using `proto.Equal` for comparison.

Reply:
If I print the pointer itself I am able to bypass the govet issue

```
	t.Logf("GRPC Register SAA: %#+v", regRes)
```
And the log result looks like the same (just for the & indicating it is a pointer)

With Pointer 

![image](https://user-images.githubusercontent.com/16157139/162664101-8ad1c753-01b5-479f-a7e3-62ae75f5e818.png)


without the pointer 

![image](https://user-images.githubusercontent.com/16157139/162664157-0cd7e06f-244c-4289-9d4f-12f49f24ba6b.png)